access/src/main/external/static/css/admin/jquery-ui.css

```diff
@@ -955,6 +955,7 @@ body .ui-tooltip {
 
 .ui-icon-close { background-position: -80px -128px; }
 .ui-icon-closethick { background-position: -96px -128px; }
+.ui-icon-check { background-position: -64px -144px; }
```

Comment:
This patch is just for this version of Jquery. CSS of Jquery Editable library has not been updated.